mstp.c

```diff
@@ -2876,8 +2880,6 @@ static void PTSM_tick(port_t *prt)
         --(prt->helloWhen);
     if(prt->mdelayWhile)
         --(prt->mdelayWhile);
-    if(prt->edgeDelayWhile)
```

Comment:
Taking edgeDelayWhile out of the state machine `tick` seems a little weird to me (will elaborate in summary)